src/corelibs/U2View/src/phyltree/CreatePhyTreeDialogController.cpp

```diff
@@ -171,7 +172,13 @@ bool CreatePhyTreeDialogController::checkMemory() {
 
 void CreatePhyTreeDialogController::initSaveController(const MultipleSequenceAlignmentObject *msaObject) {
     SaveDocumentControllerConfig config;
-    config.defaultFileName = GUrlUtils::getNewLocalUrlByExtention(msaObject->getDocument()->getURLString(), msaObject->getGObjectName(), ".nwk", "");
+    QString urlStr;
+    if (FileAndDirectoryUtils::isDirectoryWritable(QFileInfo(msaObject->getDocument()->getURLString()).dir().path())) {
```

Comment:
Please cache "msaObject->getDocument()->getURLString()" into a local variable to make code more readable